src/net.cpp

```diff
@@ -746,15 +746,15 @@ std::optional<CNetMessage> V1TransportDeserializer::GetMessage(const std::chrono
 
     // Check checksum and header command string
     if (memcmp(hash.begin(), hdr.pchChecksum, CMessageHeader::CHECKSUM_SIZE) != 0) {
-        LogPrint(BCLog::NET, "CHECKSUM ERROR (%s, %u bytes), expected %s was %s, peer=%d\n",
+        LogPrint(BCLog::NET, "Header error: Wrong checksum (%s, %u bytes), expected %s was %s, peer=%d\n",
                  SanitizeString(msg->m_command), msg->m_message_size,
                  HexStr(Span<uint8_t>(hash.begin(), hash.begin() + CMessageHeader::CHECKSUM_SIZE)),
                  HexStr(hdr.pchChecksum),
                  m_node_id);
         out_err_raw_size = msg->m_raw_message_size;
         msg = std::nullopt;
     } else if (!hdr.IsCommandValid()) {
-        LogPrint(BCLog::NET, "HEADER ERROR - COMMAND (%s, %u bytes), peer=%d\n",
+        LogPrint(BCLog::NET, "Header error: Invalid command (%s, %u bytes), peer=%d\n",
```

Comment:
3f0ad69c8bf4b63c6f28753191f540c4722fe8b3: I think the reason that all this used upper case was that COMMAND was initially picked as log message and to not break grepping the debug log, it was kept this way. Since you are breaking `grep COMMAND` here, might as well change it to `message type` to better reflect the meaning?

Reply:
Agree, 'command' means nothing in a P2P network that doesn't even have ad-hoc hierarchy, will change it to message type.